pkg/kv/kvserver/logstore/logstore.go

```diff
@@ -496,9 +496,9 @@ func LoadEntries(
 	eCache *raftentry.Cache,
 	sideloaded SideloadStorage,
 	lo, hi, maxBytes uint64,
-) ([]raftpb.Entry, error) {
+) (_ []raftpb.Entry, _cachedSize int, _loadedSize int, _ error) {
```

Comment:
Return `uint64` or `int64` to avoid int downcasts below?

Reply:
Done.